lib/clients.ex

```diff
@@ -0,0 +1,33 @@
+defmodule GlassFactoryApi.Clients do
+  @moduledoc """
+  Provides the methods to list clients of an organization.
+  """
+  alias GlassFactoryApi.ApiClient
+  alias GlassFactoryApi.Clients.Client
+
+  @doc """
+  Get a client given the id
+
+  ## Examples
+
+      iex> GlassFactoryApi.Clients.get_client("1234")
+      %Client{
+        id: 1234,
+        name: "Google",
+        archived_at: null,
+        owner_id: 567,
+        office_id: 789
+      }
+  """
+  @spec get_client(integer(), map) :: nil | GlassFactoryApi.Clients.Client.t()
+  def get_client(client_id, config \\ %{}) do
```

Comment:
After a pair programming with @britto, he suggested me to create two versions of the resources methods, one with and another without bang.  

This way we can have a safer method which returns a tuple with `{:ok, resource}` and in a fail case returns {:error, message}`, and the banged one returns the resource directly or raises in case of fail.

Would you mind implement `get_client!\2`too, please? 😄

Reply:
Sure my friend! 😄